app/change_sets/hyrax/dynamic_change_set.rb

```diff
@@ -3,12 +3,12 @@
 module Hyrax
   class DynamicChangeSet
     def self.new(obj, *args)
-      change_set_class = if obj.is_a? Collection
-                           "Hyrax::#{obj.class}ChangeSet"
-                         else
-                           "#{obj.class}ChangeSet"
+      change_set_class = begin
+                           "::#{obj.class}ChangeSet".constantize
+                         rescue NameError
+                           "Hyrax::#{obj.class}ChangeSet".constantize
```

Comment:
Does this case ever get used anymore?

Reply:
This solves the problem with FileSet not being namespaced, ex:

```
[9] pry(#<Hyrax::ChangeSetPersister>)> metadata_adapter.query_service.find_members(resource: change_set.resource)[1].class
=> FileSet
[10] pry(#<Hyrax::ChangeSetPersister>)> metadata_adapter.query_service.find_members(resource: change_set.resource)[1].is_a?(FileSet)
=> false
[11] pry(#<Hyrax::ChangeSetPersister>)> metadata_adapter.query_service.find_members(resource: change_set.resource)[1].is_a?(::FileSet)
=> true
```

Reply:
I don't think there are any namespaced changesets anymore (for things that use DynamicChangeSet)

Reply:
Is there a reason we are ever allowing an instance of FileSet as a valid parameter to DynamicChangeSet#new?